dbms/src/Core/SpillHandler.cpp

```diff
@@ -63,24 +76,37 @@ void SpillHandler::spillBlocks(const Blocks & blocks)
     try
```

Comment:
Better add random fail point here to check.

Reply:
Done